requirements/edx/github.txt

```diff
@@ -89,7 +89,7 @@ git+https://github.com/edx/edx-val.git@0.0.8#egg=edxval==0.0.8
 -e git+https://github.com/edx/edx-search.git@release-2015-11-17#egg=edx-search==0.1.1
 -e git+https://github.com/edx/edx-milestones.git@release-2015-11-17#egg=edx-milestones==0.1.5
 git+https://github.com/edx/edx-lint.git@v0.3.2#egg=edx_lint==0.3.2
-git+https://github.com/edx/xblock-utils.git@v1.0.0#egg=xblock-utils==v1.0.0
+git+https://github.com/edx/xblock-utils.git@v1.0.1#egg=xblock-utils==v1.0.1
```

Comment:
The v1.0.1 tag in xblock-utils has a setup.py that says version="1.0.0".  Also, it's more usual to leave the "v" out of the version number in setup.py, and in the last component of this URL.